css/index.css

```
@@ -1,3 +1,177 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
 
+.container {
+    margin: 10px auto;
+    max-width: 876px;
+    padding-top: 20px;
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+   header {
+    margin-bottom: 100px;
+  }
+   
+  nav {
+    margin-bottom: 40px;
+    
+  }
+   nav a {
+    text-decoration: none;
+    color: darkgray;
+    margin-right: 50px;
+    
+  }
+   nav a:hover {
+    color: black;
+  }
+   nav {
+    display: flex;
+    align-items: baseline;
+  }
+   .logo {
+    margin-left: 80px;
+    padding-top: 10px;
+  }
+    
+   .tagline-section {
+    display: flex;
+    margin-top: 35px;
+    margin-bottom: 70px;
+    padding-bottom: 40px;
+    border-bottom: 1px solid black;
+  }
+   .tagline-section > div {
+    width: 50%;
+    text-align: center;
+   }
+   .tagline-section h1 {
+    font-size: 82px;
+    text-align: center;
+    margin-top: 20px;
+  }
+   .tagline-section img {
+    padding-left: 50px;
+  }
+   .button {
+    border: 1px solid black;
+    padding: 10px;
+    color: black;
+    background-color: white;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    text-align: center;
+    width: 170px;
+    cursor: pointer;
+  }
+   button:hover {
+    color: white;
+    background-color: black;
+  }
+   .middle-img {
+    width: 100%;
+    height: auto;
+  }
+   
+   .top-content {
+    display: flex;
+    margin-bottom: 40px;
+    margin-top: 40px;
+  }
+   .top-content > div {
+    max-width: 450px;
+    padding-right: 20px;
+  }
+   
+   .bottom-content {
+    display: flex;
+    padding-left: 10px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+    
+  }
+   .bottom-content > div {
+    max-width: 300px;
+    padding-right: 20px;
+    
+  }
+   address {
+    margin-top: 40px;
+  }
+  footer {
+    margin-top: 50px;
+    text-align: center;
+    margin-bottom: 50px;
+  }
+   
+  p {
+    margin-top: 1em;
+    margin-bottom: 1em;
+    margin-left: 0;
+    margin-right: 0;
+  }
+  
+   h2 {
+    font-weight: bold;
+  }
+   @media  (max-width: 800px) {
+     header {
+      margin-bottom: 0px;
+    }
+    .logo {
+      margin-left: 0;
+      padding-top: 0;
+    }
+     .tagline-section {
+      display: none;
+    }
+     .tagline-section {
+      justify-content: center;
+    } 
+
+   @media (max-width: 500px) {
+    nav {
+      padding-left: 0px;
+    }
+    
```

```suggestion
        nav {
      flex-direction: column;
      align-items: center;
      max-width: 500px;
    }
    nav {
      
      width: 100%;
      text-align: center;
      padding: 20px;
    }
``` might want to merge these